packages/test/mocha-test-setup/mocharc-common.js

```diff
@@ -11,6 +11,8 @@ const path = require("path");
 function getFluidTestMochaConfig(packageDir, additionalRequiredModules) {
 
     const testDriver = process.env.fluid__test__driver ? process.env.fluid__test__driver : "local";
+    const r11sEndpointName = process.env.fluid__test__r11sEndpointName;
+    const testVariant = (testDriver === "r11s" || testDriver === "routerlicious") && (r11sEndpointName !== "r11s")? `r11s-${r11sEndpointName}` : testDriver;
```

Comment:
is there anything preventing us from migrating entirely to r11s and t9s across the board and not making these double checks?

Reply:
@heliocliu  Probably not.   I hate the double check as well.  So feel free to do it if I don't get to it first.